owncloudComLib/src/main/java/lib/resources/status/GetRemoteStatusOperation.java

```diff
@@ -52,7 +52,7 @@
  * @author David González Verdugo
  */
 
-public class GetRemoteStatusOperation extends RemoteOperation<OwnCloudVersion> {
+public class GetRemoteStatusOperation extends RemoteOperation<com.owncloud.android.lib.resources.status.OwnCloudVersion> {
```

Comment:
Think `com.owncloud.android.lib` doesn't exist anymore, are you sure this is properly compiling?

Reply:
Anyway, what are you using to do this migration? Any automatic tool?

Reply:
Android Studio does it during move of class locations

Reply:
> Think `com.owncloud.android.lib` doesn't exist anymore, are you sure this is properly compiling?

https://travis-ci.org/owncloud/android-library/builds/467152043?utm_source=github_status&utm_medium=notification